rust/src/participant.rs

```diff
@@ -113,7 +115,17 @@ impl Participant {
 
     /// Compose an update message.
     pub fn compose_update_message(&self, pk: &CoordinatorPublicKey, sum_dict: &SumDict) -> Vec<u8> {
-        let (mask_seed, masked_model) = Self::mask_model();
+        let model = Model::try_from(vec![0_f32, 0.5, -0.5]).unwrap(); // dummy
```

Comment:
Is this for testing or should we hardcode this for now?

Reply:
it's mainly for integrating the masking, since it needs a (dummy) model to operate on. i would leave it in for now, i guess it's going to be replaced soon anyways when Kwok finishes his work on the participant.

Reply:
sounds good